app/uk/gov/hmrc/customs/declaration/services/declarationServices.scala

```diff
@@ -173,8 +173,8 @@ trait ApiSubscriptionFieldsService {
             if (response.fields.authenticatedEori.exists(_.trim.nonEmpty)) {
               Right(response)
             } else {
-              logger.error(s"authenticatedEori for CSP not returned from api subscription fields for client id: ${c.value}")
```

Comment:
Why has the clientId been removed from the log message? I always find it useful to include specifics in log messages when trying to debug issues.

Reply:
Because it's included implicitly in the logging